src/components/interfaces/MOBILE_API.xml

```diff
@@ -3352,9 +3352,9 @@
             Indicates if the light's density can be set remotely (similar to a dimmer).
             </description>
         </param>
-        <param name="sRGBColorSpaceAvailable" type="Boolean" mandatory="false">
+        <param name="RGBColorSpaceAvailable" type="Boolean" mandatory="false">
```

Comment:
this shall be `rgbColorAvailable`

Reply:
According to the proposal https://github.com/smartdevicelink/sdl_evolution/blob/b1ced0e8be3a8b10adae6bf17904f2ce116d64ad/proposals/0099-new-remote-control-modules-and-parameters.md  the `RGB` should be uppercase

Reply:
@LuxoftAKutsan @joeljfischer I think that's a typo in the proposal PR. The parameters shall start with a lower case letter.

Reply:
or `colorAvailable`

Reply:
According to the comments here: https://github.com/smartdevicelink/sdl_evolution/pull/572 it should be rgbColorAvailable.  

>I just noticed I named an RPC parameter RGBColorSpaceAvailable, when it should be >rgbColorSpaceAvailable.

@joeljfischer can you please confirm? thanks

Reply:
@yang1070 fixed in c849edd46a. Please clarify, should we remove this fix, or we will fix the proposal?

Reply:
Please note the difference between rgbColorSpaceAvailable and rgbColorAvailable.  We need to make sure the value is correct.

Reply:
@LuxoftAKutsan @mrapitis @joeljfischer  this shall be `rgbColorSpaceAvailable`